clientd3d/music.c

```diff
@@ -132,9 +136,11 @@ void MusicClose(void)
 //	Not used by MSS version
 /*
  * OpenMidiFile:  Open midi file for playing.
+ *   For background room music opens midi_bg_music_element device.
+ *   For any other music (jala) opens midi_element device.
```

Comment:
You should generally avoid referring to game-specific terms like "Jala" or even "room" in files like this.  Suppose someone adds another spell school one day... does this function not work, because it says it only works with Jala music?  You get the idea.  The client doesn't know about game details.

Reply:
I've fixed this everywhere.  Jala is mentioned in one comment still where I think it is an appropriate example for future coders.